dashboard/src/components/modal/Modal.stories.tsx

```diff
@@ -0,0 +1,109 @@
+// Modal.stories.ts|tsx
+
+import { ComponentStory, ComponentMeta } from "@storybook/react";
```

Comment:
I wasn't able to get the storybook running.

<img width="1306" alt="image" src="https://user-images.githubusercontent.com/127234894/225804985-c7dd9ab1-0ac3-49d0-9075-9fd6867c60b5.png">

Reply:
@ErezBiren other then that amazing job!